[2025-07-22T10:18:50.602+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-22T10:18:50.648+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres.load_user_dim_table manual__2025-07-22T10:18:42.652724+00:00 [queued]>
[2025-07-22T10:18:50.677+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres.load_user_dim_table manual__2025-07-22T10:18:42.652724+00:00 [queued]>
[2025-07-22T10:18:50.679+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-22T10:18:50.718+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_user_dim_table> on 2025-07-22 10:18:42.652724+00:00
[2025-07-22T10:18:50.742+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres', 'load_user_dim_table', 'manual__2025-07-22T10:18:42.652724+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpwcqogfpo']
[2025-07-22T10:18:50.748+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1555) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-22T10:18:50.758+0000] {standard_task_runner.py:105} INFO - Job 64: Subtask load_user_dim_table
[2025-07-22T10:18:50.763+0000] {standard_task_runner.py:72} INFO - Started process 1559 to run task
[2025-07-22T10:18:50.960+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres.load_user_dim_table manual__2025-07-22T10:18:42.652724+00:00 [running]> on host 0c6a6447a272
[2025-07-22T10:18:51.144+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres' AIRFLOW_CTX_TASK_ID='load_user_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-22T10:18:42.652724+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-22T10:18:42.652724+00:00'
[2025-07-22T10:18:51.146+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-22T10:18:51.652+0000] {etl_dag.py:63} INFO - Loading user dimension table
[2025-07-22T10:18:51.659+0000] {etl_dag.py:66} INFO - User dimension table loaded successfully
[2025-07-22T10:18:51.661+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-22T10:18:51.690+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-22T10:18:51.692+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres, task_id=load_user_dim_table, run_id=manual__2025-07-22T10:18:42.652724+00:00, execution_date=20250722T101842, start_date=20250722T101850, end_date=20250722T101851
[2025-07-22T10:18:51.756+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-22T10:18:51.775+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
